registry/api/errcode/errors.go

```diff
@@ -97,25 +101,28 @@ type Error struct {
 
 var _ error = Error{}
 
-// ErrorCode returns the ID/Value of this Error
+// ErrorCode returns the ID/Value of this Error.
 func (e Error) ErrorCode() ErrorCode {
 	return e.Code
 }
 
 // Error returns a human readable representation of the error.
 func (e Error) Error() string {
-	return fmt.Sprintf("%s: %s",
```

Comment:
@stevvooe One option is to add a `Error.SetErrorTemplate()` method which would allow users of this package to define the format of the Error() text string.  The 'distribution' could make it `code:text` while the engine could make it just `text`.  Same for ErrCode object too.